include/fmt/args.h

```diff
@@ -143,6 +143,38 @@ class dynamic_format_arg_store
   }
 
  public:
+  /**
+    \rst
+    Default constructor of `dynamic_format_arg_store`
+    \endrst
+  */
+  constexpr FMT_INLINE dynamic_format_arg_store() = default;
+
+  /**
+    \rst
+    Constructs a `dynamic_format_arg_store` object from
+    `dynamic_format_arg_store`.
+
+    **Example**::
+
+      fmt::dynamic_format_arg_store<fmt::format_context> store;
+      store.push_back(123);
+      store.push_back("test");
+
+      fmt::dynamic_format_arg_store<fmt::format_context> store2(store);
+    \endrst
+  */
+  constexpr FMT_INLINE dynamic_format_arg_store(
+      const dynamic_format_arg_store<Context>& store)
+      : data_(store.data_),
+        named_info_(store.named_info_)
+#if FMT_GCC_VERSION && FMT_GCC_VERSION < 409
+        ,
+        basic_format_args<Context>()
+#endif
```

Comment:
Why is this necessary?

Reply:
this was necessary(?) because it way saying that the copy constructor needs to initialize the parent class too.

Reply:
I suggest moving this to the beginning of the initialization list to improve formatting and since base classes are normally initialized first.